nemo/README.md

```diff
@@ -82,22 +84,22 @@ The model will be (downloaded if required and) loaded in memory when calling the
 
 If you want to preload the model (and later specify a path `<ASR_PATH>` as `MODEL`),
 you may want to download one of NeMo models:
-   * [French large fast conformer with punctuations made by NVIDIA](https://huggingface.co/nvidia/stt_fr_fastconformer_hybrid_large_pc)
-   * [French large fast conformer made by Bofeng Huang](https://huggingface.co/bofenghuang/stt_fr_fastconformer_hybrid_large)
-   * [English large fast conformer made by NVIDIA](https://huggingface.co/nvidia/stt_en_fastconformer_transducer_large)
-   * [English XL fast conformer made by NVIDIA](https://huggingface.co/nvidia/parakeet-ctc-0.6b)
-   * [English XXL fast conformer made by NVIDIA](https://huggingface.co/nvidia/parakeet-ctc-1.1b)
-   * Soon models trained by LINAGORA
+   * [LinTO French Large Fast Conformer (by LINAGORA)](https://huggingface.co/linagora/linto_stt_fr_fastconformer): Most robust French model
+   * [French Large Fast Conformer (by NVIDIA)](https://huggingface.co/nvidia/stt_fr_fastconformer_hybrid_large_pc): Includes uppercase letters and punctuation, but is less precise than the LINAGORA model
+   * [French Large Fast Conformer (by Bofeng Huang)](https://huggingface.co/bofenghuang/stt_fr_fastconformer_hybrid_large): Performs well on reading and prepared speech
+   * [English Large Fast Conformer (by NVIDIA)](https://huggingface.co/nvidia/stt_en_fastconformer_transducer_large)
+   * [English XL Fast Conformer (by NVIDIA)](https://huggingface.co/nvidia/parakeet-ctc-0.6b)
+   * [English XXL Fast Conformer (by NVIDIA)](https://huggingface.co/nvidia/parakeet-ctc-1.1b)
```

Comment:
Wouln't it be great to indicate what kind of model it is, for each one of them ? (rnnt_bpe, ...)